corehq/blobs/pillow.py

```diff
@@ -19,46 +14,28 @@
 
 class BlobDeletionProcessor(PillowProcessor):
 
-    def __init__(self, blob_db, bucket_base):
+    def __init__(self, blob_db):
         super(BlobDeletionProcessor, self).__init__()
         self.blob_db = blob_db
-        self.bucket_base = bucket_base
 
     def process_change(self, pillow_instance, change):
         if change.deleted:
-            bucket = join(self.bucket_base, change.id)
+            bucket = "{}/{}".format(change.meta.data_source_name, change.id)
```

Comment:
Cool, thanks for looking into it.

There is another proposal I intend to follow up on at some point that would render this irrelevant: make the blob db append-only (never delete blobs). The hope is to make it unnecessary to backup the riak cluster, since it already maintains redundancy and therefore doesn't require backups as long as nothing is ever deleted. I'd like to do a few calculations to do a rough comparison of the (cost of retaining) space for deleted blobs against the cost of maintaining backups for the riak cluster.